error: 
    status - 500 
    message - Error: Book Does not exsist
    at _callee14$ (D:\Book_Store_App\Book-Store\src\services\/user.service.js:380:12)
    at tryCatch (D:\Book_Store_App\Book-Store\node_modules\regenerator-runtime\runtime.js:63:40)
    at Generator.invoke [as _invoke] (D:\Book_Store_App\Book-Store\node_modules\regenerator-runtime\runtime.js:294:22)
    at Generator.next (D:\Book_Store_App\Book-Store\node_modules\regenerator-runtime\runtime.js:119:21)
    at asyncGeneratorStep (D:\Book_Store_App\Book-Store\node_modules\@babel\runtime\helpers\asyncToGenerator.js:3:24)
    at _next (D:\Book_Store_App\Book-Store\node_modules\@babel\runtime\helpers\asyncToGenerator.js:25:9)
    at processTicksAndRejections (internal/process/task_queues.js:95:5) 
    url - /api/v1/wishlist/61f8299c20bebd4478718981 
    method - POST 
    IP - ::ffff:127.0.0.1
   {"timestamp":"2022-01-31T18:30:13.069Z"}
info: Server started at http://localhost:3000/api/v1/ {"timestamp":"2022-01-31T18:32:16.801Z"}
info: Connected to the database. {"timestamp":"2022-01-31T18:32:16.834Z"}
info: Server started at http://localhost:3000/api/v1/ {"timestamp":"2022-01-31T18:32:54.646Z"}
info: Connected to the database. {"timestamp":"2022-01-31T18:32:54.688Z"}
info: Server started at http://localhost:3000/api/v1/ {"timestamp":"2022-01-31T18:34:45.563Z"}
info: Connected to the database. {"timestamp":"2022-01-31T18:34:45.613Z"}
info: Server started at http://localhost:3000/api/v1/ {"timestamp":"2022-01-31T18:35:01.589Z"}
info: Connected to the database. {"timestamp":"2022-01-31T18:35:01.634Z"}
info: Server started at http://localhost:3000/api/v1/ {"timestamp":"2022-01-31T18:35:19.477Z"}
info: Connected to the database. {"timestamp":"2022-01-31T18:35:19.505Z"}
info: Server started at http://localhost:3000/api/v1/ {"timestamp":"2022-01-31T18:35:33.809Z"}
info: Connected to the database. {"timestamp":"2022-01-31T18:35:33.834Z"}
error: 
    status - 500 
    message - Error: Book Not Available
    at _callee10$ (D:\Book_Store_App\Book-Store\src\services\/user.service.js:268:11)
    at tryCatch (D:\Book_Store_App\Book-Store\node_modules\regenerator-runtime\runtime.js:63:40)
    at Generator.invoke [as _invoke] (D:\Book_Store_App\Book-Store\node_modules\regenerator-runtime\runtime.js:294:22)
    at Generator.next (D:\Book_Store_App\Book-Store\node_modules\regenerator-runtime\runtime.js:119:21)
    at asyncGeneratorStep (D:\Book_Store_App\Book-Store\node_modules\@babel\runtime\helpers\asyncToGenerator.js:3:24)
    at _next (D:\Book_Store_App\Book-Store\node_modules\@babel\runtime\helpers\asyncToGenerator.js:25:9)
    at processTicksAndRejections (internal/process/task_queues.js:95:5) 
    url - /api/v1/cart/ 
    method - POST 
    IP - ::1
   {"timestamp":"2022-01-31T18:38:38.793Z"}
error: 
    status - 500 
    message - Error: Book Out Of Stock Stock
    at _callee10$ (D:\Book_Store_App\Book-Store\src\services\/user.service.js:266:11)
    at tryCatch (D:\Book_Store_App\Book-Store\node_modules\regenerator-runtime\runtime.js:63:40)
    at Generator.invoke [as _invoke] (D:\Book_Store_App\Book-Store\node_modules\regenerator-runtime\runtime.js:294:22)
    at Generator.next (D:\Book_Store_App\Book-Store\node_modules\regenerator-runtime\runtime.js:119:21)
    at asyncGeneratorStep (D:\Book_Store_App\Book-Store\node_modules\@babel\runtime\helpers\asyncToGenerator.js:3:24)
    at _next (D:\Book_Store_App\Book-Store\node_modules\@babel\runtime\helpers\asyncToGenerator.js:25:9)
    at processTicksAndRejections (internal/process/task_queues.js:95:5) 
    url - /api/v1/cart/ 
    method - POST 
    IP - ::1
   {"timestamp":"2022-01-31T18:38:51.102Z"}
info: Server started at http://localhost:3000/api/v1/ {"timestamp":"2022-01-31T18:39:17.641Z"}
info: Connected to the database. {"timestamp":"2022-01-31T18:39:17.667Z"}
info: Server started at http://localhost:3000/api/v1/ {"timestamp":"2022-01-31T18:42:39.319Z"}
info: Connected to the database. {"timestamp":"2022-01-31T18:42:39.344Z"}
info: Server started at http://localhost:3000/api/v1/ {"timestamp":"2022-01-31T18:42:58.571Z"}
info: Connected to the database. {"timestamp":"2022-01-31T18:42:58.601Z"}
info: Server started at http://localhost:3000/api/v1/ {"timestamp":"2022-01-31T18:43:34.681Z"}
info: Connected to the database. {"timestamp":"2022-01-31T18:43:34.704Z"}
info: Server started at http://localhost:3000/api/v1/ {"timestamp":"2022-01-31T18:43:57.280Z"}
info: Connected to the database. {"timestamp":"2022-01-31T18:43:57.306Z"}
info: Server started at http://localhost:3000/api/v1/ {"timestamp":"2022-01-31T18:44:15.394Z"}
info: Connected to the database. {"timestamp":"2022-01-31T18:44:15.424Z"}
info: Server started at http://localhost:3000/api/v1/ {"timestamp":"2022-01-31T18:44:51.864Z"}
info: Connected to the database. {"timestamp":"2022-01-31T18:44:51.893Z"}
info: Server started at http://localhost:3000/api/v1/ {"timestamp":"2022-01-31T18:45:13.284Z"}
info: Connected to the database. {"timestamp":"2022-01-31T18:45:13.329Z"}
info: Server started at http://localhost:3000/api/v1/ {"timestamp":"2022-01-31T18:45:47.067Z"}
info: Connected to the database. {"timestamp":"2022-01-31T18:45:47.094Z"}
info: Server started at http://localhost:3000/api/v1/ {"timestamp":"2022-01-31T18:46:01.468Z"}
info: Connected to the database. {"timestamp":"2022-01-31T18:46:01.514Z"}
info: Server started at http://localhost:3000/api/v1/ {"timestamp":"2022-01-31T18:46:37.494Z"}
info: Connected to the database. {"timestamp":"2022-01-31T18:46:37.527Z"}
info: Server started at http://localhost:3000/api/v1/ {"timestamp":"2022-01-31T18:47:00.544Z"}
info: Connected to the database. {"timestamp":"2022-01-31T18:47:00.633Z"}
info: Server started at http://localhost:3000/api/v1/ {"timestamp":"2022-01-31T18:47:19.843Z"}
info: Connected to the database. {"timestamp":"2022-01-31T18:47:19.868Z"}
info: Server started at http://localhost:3000/api/v1/ {"timestamp":"2022-01-31T18:54:12.222Z"}
info: Connected to the database. {"timestamp":"2022-01-31T18:54:12.267Z"}
info: Server started at http://localhost:3000/api/v1/ {"timestamp":"2022-01-31T18:55:12.636Z"}
info: Connected to the database. {"timestamp":"2022-01-31T18:55:12.661Z"}
info: Server started at http://localhost:3000/api/v1/ {"timestamp":"2022-01-31T18:56:10.962Z"}
info: Connected to the database. {"timestamp":"2022-01-31T18:56:10.989Z"}
info: Server started at http://localhost:3000/api/v1/ {"timestamp":"2022-01-31T18:57:43.420Z"}
info: Connected to the database. {"timestamp":"2022-01-31T18:57:43.466Z"}
info: Server started at http://localhost:3000/api/v1/ {"timestamp":"2022-01-31T18:58:18.225Z"}
info: Connected to the database. {"timestamp":"2022-01-31T18:58:18.260Z"}
info: Server started at http://localhost:3000/api/v1/ {"timestamp":"2022-01-31T18:59:09.969Z"}
info: Connected to the database. {"timestamp":"2022-01-31T18:59:09.993Z"}
info: Server started at http://localhost:3000/api/v1/ {"timestamp":"2022-01-31T18:59:55.565Z"}
info: Connected to the database. {"timestamp":"2022-01-31T18:59:55.597Z"}
info: Server started at http://localhost:3000/api/v1/ {"timestamp":"2022-01-31T19:00:03.586Z"}
info: Connected to the database. {"timestamp":"2022-01-31T19:00:03.627Z"}
info: Server started at http://localhost:3000/api/v1/ {"timestamp":"2022-01-31T19:00:11.193Z"}
info: Connected to the database. {"timestamp":"2022-01-31T19:00:11.224Z"}
info: Server started at http://localhost:3000/api/v1/ {"timestamp":"2022-01-31T19:00:19.424Z"}
info: Connected to the database. {"timestamp":"2022-01-31T19:00:19.451Z"}
info: Server started at http://localhost:3000/api/v1/ {"timestamp":"2022-01-31T19:04:18.090Z"}
info: Connected to the database. {"timestamp":"2022-01-31T19:04:18.119Z"}
info: Server started at http://localhost:3000/api/v1/ {"timestamp":"2022-01-31T19:04:23.847Z"}
info: Connected to the database. {"timestamp":"2022-01-31T19:04:23.874Z"}
info: Server started at http://localhost:3000/api/v1/ {"timestamp":"2022-01-31T19:04:57.549Z"}
info: Connected to the database. {"timestamp":"2022-01-31T19:04:57.573Z"}
info: Server started at http://localhost:3000/api/v1/ {"timestamp":"2022-01-31T19:05:42.903Z"}
info: Connected to the database. {"timestamp":"2022-01-31T19:05:42.932Z"}
info: Server started at http://localhost:3000/api/v1/ {"timestamp":"2022-01-31T19:05:55.274Z"}
info: Connected to the database. {"timestamp":"2022-01-31T19:05:55.330Z"}
info: Server started at http://localhost:3000/api/v1/ {"timestamp":"2022-01-31T19:06:12.184Z"}
info: Connected to the database. {"timestamp":"2022-01-31T19:06:12.213Z"}
info: Server started at http://localhost:3000/api/v1/ {"timestamp":"2022-01-31T19:06:58.901Z"}
info: Connected to the database. {"timestamp":"2022-01-31T19:06:58.926Z"}
info: Server started at http://localhost:3000/api/v1/ {"timestamp":"2022-01-31T19:08:07.528Z"}
info: Connected to the database. {"timestamp":"2022-01-31T19:08:07.553Z"}
error: 
    status - 500 
    message - Error: Book Not in wishlist
    at _callee15$ (D:\Book_Store_App\Book-Store\src\services\/user.service.js:403:9)
    at tryCatch (D:\Book_Store_App\Book-Store\node_modules\regenerator-runtime\runtime.js:63:40)
    at Generator.invoke [as _invoke] (D:\Book_Store_App\Book-Store\node_modules\regenerator-runtime\runtime.js:294:22)
    at Generator.next (D:\Book_Store_App\Book-Store\node_modules\regenerator-runtime\runtime.js:119:21)
    at asyncGeneratorStep (D:\Book_Store_App\Book-Store\node_modules\@babel\runtime\helpers\asyncToGenerator.js:3:24)
    at _next (D:\Book_Store_App\Book-Store\node_modules\@babel\runtime\helpers\asyncToGenerator.js:25:9)
    at processTicksAndRejections (internal/process/task_queues.js:95:5) 
    url - /api/v1/wishlist/61f64042a99dbd428ca5261c 
    method - PATCH 
    IP - ::ffff:127.0.0.1
   {"timestamp":"2022-01-31T19:09:49.112Z"}
error: 
    status - 500 
    message - Error: Book Not in wishlist
    at _callee15$ (D:\Book_Store_App\Book-Store\src\services\/user.service.js:403:9)
    at tryCatch (D:\Book_Store_App\Book-Store\node_modules\regenerator-runtime\runtime.js:63:40)
    at Generator.invoke [as _invoke] (D:\Book_Store_App\Book-Store\node_modules\regenerator-runtime\runtime.js:294:22)
    at Generator.next (D:\Book_Store_App\Book-Store\node_modules\regenerator-runtime\runtime.js:119:21)
    at asyncGeneratorStep (D:\Book_Store_App\Book-Store\node_modules\@babel\runtime\helpers\asyncToGenerator.js:3:24)
    at _next (D:\Book_Store_App\Book-Store\node_modules\@babel\runtime\helpers\asyncToGenerator.js:25:9)
    at processTicksAndRejections (internal/process/task_queues.js:95:5) 
    url - /api/v1/wishlist/61f64054a99dbd428ca5261e 
    method - PATCH 
    IP - ::ffff:127.0.0.1
   {"timestamp":"2022-01-31T19:10:37.169Z"}
info: Server started at http://localhost:3000/api/v1/ {"timestamp":"2022-01-31T19:11:24.328Z"}
info: Connected to the database. {"timestamp":"2022-01-31T19:11:24.357Z"}
info: Server started at http://localhost:3000/api/v1/ {"timestamp":"2022-01-31T19:11:30.233Z"}
info: Connected to the database. {"timestamp":"2022-01-31T19:11:30.257Z"}
info: Server started at http://localhost:3000/api/v1/ {"timestamp":"2022-02-01T02:59:43.606Z"}
info: Connected to the database. {"timestamp":"2022-02-01T02:59:43.642Z"}
error: 
    status - 500 
    message - Error: Book Not in wishlist
    at _callee15$ (D:\Book_Store_App\Book-Store\src\services\/user.service.js:404:9)
    at tryCatch (D:\Book_Store_App\Book-Store\node_modules\regenerator-runtime\runtime.js:63:40)
    at Generator.invoke [as _invoke] (D:\Book_Store_App\Book-Store\node_modules\regenerator-runtime\runtime.js:294:22)
    at Generator.next (D:\Book_Store_App\Book-Store\node_modules\regenerator-runtime\runtime.js:119:21)
    at asyncGeneratorStep (D:\Book_Store_App\Book-Store\node_modules\@babel\runtime\helpers\asyncToGenerator.js:3:24)
    at _next (D:\Book_Store_App\Book-Store\node_modules\@babel\runtime\helpers\asyncToGenerator.js:25:9)
    at processTicksAndRejections (internal/process/task_queues.js:95:5) 
    url - /api/v1/wishlist/61f64042a99dbd428ca5261c 
    method - PATCH 
    IP - ::ffff:127.0.0.1
   {"timestamp":"2022-02-01T03:10:51.966Z"}
info: Server started at http://localhost:3000/api/v1/ {"timestamp":"2022-02-01T03:38:11.018Z"}
info: Connected to the database. {"timestamp":"2022-02-01T03:38:11.047Z"}
error: 
    status - 500 
    message - Error: Already Added in Wishlist
    at _callee14$ (D:\Book_Store_App\Book-Store\src\services\/user.service.js:373:15)
    at tryCatch (D:\Book_Store_App\Book-Store\node_modules\regenerator-runtime\runtime.js:63:40)
    at Generator.invoke [as _invoke] (D:\Book_Store_App\Book-Store\node_modules\regenerator-runtime\runtime.js:294:22)
    at Generator.next (D:\Book_Store_App\Book-Store\node_modules\regenerator-runtime\runtime.js:119:21)
    at asyncGeneratorStep (D:\Book_Store_App\Book-Store\node_modules\@babel\runtime\helpers\asyncToGenerator.js:3:24)
    at _next (D:\Book_Store_App\Book-Store\node_modules\@babel\runtime\helpers\asyncToGenerator.js:25:9)
    at processTicksAndRejections (internal/process/task_queues.js:95:5) 
    url - /api/v1/wishlist/61f64042a99dbd428ca5261c 
    method - POST 
    IP - ::ffff:127.0.0.1
   {"timestamp":"2022-02-01T03:38:16.046Z"}
info: Server started at http://localhost:3000/api/v1/ {"timestamp":"2022-02-01T05:32:41.541Z"}
info: Connected to the database. {"timestamp":"2022-02-01T05:32:41.569Z"}
error: 
    status - 500 
    message - ReferenceError: req is not defined
    at _callee16$ (D:\Book_Store_App\Book-Store\src\services\/user.service.js:418:13)
    at tryCatch (D:\Book_Store_App\Book-Store\node_modules\regenerator-runtime\runtime.js:63:40)
    at Generator.invoke [as _invoke] (D:\Book_Store_App\Book-Store\node_modules\regenerator-runtime\runtime.js:294:22)
    at Generator.next (D:\Book_Store_App\Book-Store\node_modules\regenerator-runtime\runtime.js:119:21)
    at asyncGeneratorStep (D:\Book_Store_App\Book-Store\node_modules\@babel\runtime\helpers\asyncToGenerator.js:3:24)
    at _next (D:\Book_Store_App\Book-Store\node_modules\@babel\runtime\helpers\asyncToGenerator.js:25:9)
    at D:\Book_Store_App\Book-Store\node_modules\@babel\runtime\helpers\asyncToGenerator.js:32:7
    at new Promise (<anonymous>)
    at Object.<anonymous> (D:\Book_Store_App\Book-Store\node_modules\@babel\runtime\helpers\asyncToGenerator.js:21:12)
    at Object.fetchWishList (D:\Book_Store_App\Book-Store\src\services\/user.service.js:415:27) 
    url - /api/v1/wishlist/ 
    method - GET 
    IP - ::ffff:127.0.0.1
   {"timestamp":"2022-02-01T05:32:43.533Z"}
info: Server started at http://localhost:3000/api/v1/ {"timestamp":"2022-02-01T05:33:08.927Z"}
info: Connected to the database. {"timestamp":"2022-02-01T05:33:08.958Z"}
error: 
    status - 500 
    message - Error: Wishlist does not created
    at _callee16$ (D:\Book_Store_App\Book-Store\src\services\/user.service.js:423:12)
    at tryCatch (D:\Book_Store_App\Book-Store\node_modules\regenerator-runtime\runtime.js:63:40)
    at Generator.invoke [as _invoke] (D:\Book_Store_App\Book-Store\node_modules\regenerator-runtime\runtime.js:294:22)
    at Generator.next (D:\Book_Store_App\Book-Store\node_modules\regenerator-runtime\runtime.js:119:21)
    at asyncGeneratorStep (D:\Book_Store_App\Book-Store\node_modules\@babel\runtime\helpers\asyncToGenerator.js:3:24)
    at _next (D:\Book_Store_App\Book-Store\node_modules\@babel\runtime\helpers\asyncToGenerator.js:25:9)
    at processTicksAndRejections (internal/process/task_queues.js:95:5) 
    url - /api/v1/wishlist/ 
    method - GET 
    IP - ::ffff:127.0.0.1
   {"timestamp":"2022-02-01T05:34:06.961Z"}
info: Server started at http://localhost:3000/api/v1/ {"timestamp":"2022-02-01T05:37:06.209Z"}
info: Connected to the database. {"timestamp":"2022-02-01T05:37:06.274Z"}
info: Server started at http://localhost:3000/api/v1/ {"timestamp":"2022-02-01T05:37:14.545Z"}
info: Connected to the database. {"timestamp":"2022-02-01T05:37:14.585Z"}
info: Server started at http://localhost:3000/api/v1/ {"timestamp":"2022-02-01T05:37:23.481Z"}
info: Connected to the database. {"timestamp":"2022-02-01T05:37:23.519Z"}
info: Server started at http://localhost:3000/api/v1/ {"timestamp":"2022-02-01T05:37:35.629Z"}
info: Connected to the database. {"timestamp":"2022-02-01T05:37:35.667Z"}
info: Server started at http://localhost:3000/api/v1/ {"timestamp":"2022-02-01T05:38:21.514Z"}
info: Connected to the database. {"timestamp":"2022-02-01T05:38:21.555Z"}
error: 
    status - 500 
    message - Error: Wishlist does not created
    at _callee16$ (D:\Book_Store_App\Book-Store\src\services\/user.service.js:422:12)
    at tryCatch (D:\Book_Store_App\Book-Store\node_modules\regenerator-runtime\runtime.js:63:40)
    at Generator.invoke [as _invoke] (D:\Book_Store_App\Book-Store\node_modules\regenerator-runtime\runtime.js:294:22)
    at Generator.next (D:\Book_Store_App\Book-Store\node_modules\regenerator-runtime\runtime.js:119:21)
    at asyncGeneratorStep (D:\Book_Store_App\Book-Store\node_modules\@babel\runtime\helpers\asyncToGenerator.js:3:24)
    at _next (D:\Book_Store_App\Book-Store\node_modules\@babel\runtime\helpers\asyncToGenerator.js:25:9)
    at processTicksAndRejections (internal/process/task_queues.js:95:5) 
    url - /api/v1/wishlist/ 
    method - GET 
    IP - ::ffff:127.0.0.1
   {"timestamp":"2022-02-01T05:38:53.333Z"}
error: 
    status - 500 
    message - Error: Wishlist does not created
    at _callee16$ (D:\Book_Store_App\Book-Store\src\services\/user.service.js:422:12)
    at tryCatch (D:\Book_Store_App\Book-Store\node_modules\regenerator-runtime\runtime.js:63:40)
    at Generator.invoke [as _invoke] (D:\Book_Store_App\Book-Store\node_modules\regenerator-runtime\runtime.js:294:22)
    at Generator.next (D:\Book_Store_App\Book-Store\node_modules\regenerator-runtime\runtime.js:119:21)
    at asyncGeneratorStep (D:\Book_Store_App\Book-Store\node_modules\@babel\runtime\helpers\asyncToGenerator.js:3:24)
    at _next (D:\Book_Store_App\Book-Store\node_modules\@babel\runtime\helpers\asyncToGenerator.js:25:9)
    at processTicksAndRejections (internal/process/task_queues.js:95:5) 
    url - /api/v1/wishlist/ 
    method - GET 
    IP - ::ffff:127.0.0.1
   {"timestamp":"2022-02-01T05:38:54.753Z"}
error: 
    status - 500 
    message - Error: Already Added in Wishlist
    at _callee14$ (D:\Book_Store_App\Book-Store\src\services\/user.service.js:372:15)
    at tryCatch (D:\Book_Store_App\Book-Store\node_modules\regenerator-runtime\runtime.js:63:40)
    at Generator.invoke [as _invoke] (D:\Book_Store_App\Book-Store\node_modules\regenerator-runtime\runtime.js:294:22)
    at Generator.next (D:\Book_Store_App\Book-Store\node_modules\regenerator-runtime\runtime.js:119:21)
    at asyncGeneratorStep (D:\Book_Store_App\Book-Store\node_modules\@babel\runtime\helpers\asyncToGenerator.js:3:24)
    at _next (D:\Book_Store_App\Book-Store\node_modules\@babel\runtime\helpers\asyncToGenerator.js:25:9)
    at processTicksAndRejections (internal/process/task_queues.js:95:5) 
    url - /api/v1/wishlist/61f64042a99dbd428ca5261c 
    method - POST 
    IP - ::ffff:127.0.0.1
   {"timestamp":"2022-02-01T05:39:54.905Z"}
error: 
    status - 500 
    message - Error: Already Added in Wishlist
    at _callee14$ (D:\Book_Store_App\Book-Store\src\services\/user.service.js:372:15)
    at tryCatch (D:\Book_Store_App\Book-Store\node_modules\regenerator-runtime\runtime.js:63:40)
    at Generator.invoke [as _invoke] (D:\Book_Store_App\Book-Store\node_modules\regenerator-runtime\runtime.js:294:22)
    at Generator.next (D:\Book_Store_App\Book-Store\node_modules\regenerator-runtime\runtime.js:119:21)
    at asyncGeneratorStep (D:\Book_Store_App\Book-Store\node_modules\@babel\runtime\helpers\asyncToGenerator.js:3:24)
    at _next (D:\Book_Store_App\Book-Store\node_modules\@babel\runtime\helpers\asyncToGenerator.js:25:9)
    at processTicksAndRejections (internal/process/task_queues.js:95:5) 
    url - /api/v1/wishlist/61f64042a99dbd428ca5261c 
    method - POST 
    IP - ::ffff:127.0.0.1
   {"timestamp":"2022-02-01T05:39:55.084Z"}
info: Server started at http://localhost:3000/api/v1/ {"timestamp":"2022-02-01T05:41:25.725Z"}
info: Connected to the database. {"timestamp":"2022-02-01T05:41:25.751Z"}
info: Server started at http://localhost:3000/api/v1/ {"timestamp":"2022-02-01T05:41:45.463Z"}
info: Connected to the database. {"timestamp":"2022-02-01T05:41:45.488Z"}
error: 
    status - 500 
    message - ReferenceError: decoded is not defined
    at _callee$ (D:\Book_Store_App\Book-Store\src\middlewares\/auth.middleware.js:28:28)
    at tryCatch (D:\Book_Store_App\Book-Store\node_modules\regenerator-runtime\runtime.js:63:40)
    at Generator.invoke [as _invoke] (D:\Book_Store_App\Book-Store\node_modules\regenerator-runtime\runtime.js:294:22)
    at Generator.next (D:\Book_Store_App\Book-Store\node_modules\regenerator-runtime\runtime.js:119:21)
    at asyncGeneratorStep (D:\Book_Store_App\Book-Store\node_modules\@babel\runtime\helpers\asyncToGenerator.js:3:24)
    at _next (D:\Book_Store_App\Book-Store\node_modules\@babel\runtime\helpers\asyncToGenerator.js:25:9)
    at D:\Book_Store_App\Book-Store\node_modules\@babel\runtime\helpers\asyncToGenerator.js:32:7
    at new Promise (<anonymous>)
    at D:\Book_Store_App\Book-Store\node_modules\@babel\runtime\helpers\asyncToGenerator.js:21:12
    at D:\Book_Store_App\Book-Store\src\middlewares\/auth.middleware.js:15:3 
    url - /api/v1/wishlist/ 
    method - GET 
    IP - ::ffff:127.0.0.1
   {"timestamp":"2022-02-01T05:41:50.112Z"}
info: Server started at http://localhost:3000/api/v1/ {"timestamp":"2022-02-01T05:47:15.411Z"}
info: Connected to the database. {"timestamp":"2022-02-01T05:47:15.442Z"}
info: Server started at http://localhost:3000/api/v1/ {"timestamp":"2022-02-01T05:48:05.922Z"}
info: Connected to the database. {"timestamp":"2022-02-01T05:48:05.946Z"}
info: Server started at http://localhost:3000/api/v1/ {"timestamp":"2022-02-01T05:48:27.804Z"}
info: Connected to the database. {"timestamp":"2022-02-01T05:48:27.830Z"}
error: 
    status - 500 
    message - ReferenceError: decoded is not defined
    at _callee$ (D:\Book_Store_App\Book-Store\src\middlewares\/auth.middleware.js:28:28)
    at tryCatch (D:\Book_Store_App\Book-Store\node_modules\regenerator-runtime\runtime.js:63:40)
    at Generator.invoke [as _invoke] (D:\Book_Store_App\Book-Store\node_modules\regenerator-runtime\runtime.js:294:22)
    at Generator.next (D:\Book_Store_App\Book-Store\node_modules\regenerator-runtime\runtime.js:119:21)
    at asyncGeneratorStep (D:\Book_Store_App\Book-Store\node_modules\@babel\runtime\helpers\asyncToGenerator.js:3:24)
    at _next (D:\Book_Store_App\Book-Store\node_modules\@babel\runtime\helpers\asyncToGenerator.js:25:9)
    at D:\Book_Store_App\Book-Store\node_modules\@babel\runtime\helpers\asyncToGenerator.js:32:7
    at new Promise (<anonymous>)
    at D:\Book_Store_App\Book-Store\node_modules\@babel\runtime\helpers\asyncToGenerator.js:21:12
    at D:\Book_Store_App\Book-Store\src\middlewares\/auth.middleware.js:15:3 
    url - /api/v1/wishlist/ 
    method - GET 
    IP - ::ffff:127.0.0.1
   {"timestamp":"2022-02-01T05:48:31.466Z"}
error: 
    status - 500 
    message - ReferenceError: decoded is not defined
    at _callee$ (D:\Book_Store_App\Book-Store\src\middlewares\/auth.middleware.js:28:28)
    at tryCatch (D:\Book_Store_App\Book-Store\node_modules\regenerator-runtime\runtime.js:63:40)
    at Generator.invoke [as _invoke] (D:\Book_Store_App\Book-Store\node_modules\regenerator-runtime\runtime.js:294:22)
    at Generator.next (D:\Book_Store_App\Book-Store\node_modules\regenerator-runtime\runtime.js:119:21)
    at asyncGeneratorStep (D:\Book_Store_App\Book-Store\node_modules\@babel\runtime\helpers\asyncToGenerator.js:3:24)
    at _next (D:\Book_Store_App\Book-Store\node_modules\@babel\runtime\helpers\asyncToGenerator.js:25:9)
    at D:\Book_Store_App\Book-Store\node_modules\@babel\runtime\helpers\asyncToGenerator.js:32:7
    at new Promise (<anonymous>)
    at D:\Book_Store_App\Book-Store\node_modules\@babel\runtime\helpers\asyncToGenerator.js:21:12
    at D:\Book_Store_App\Book-Store\src\middlewares\/auth.middleware.js:15:3 
    url - /api/v1/wishlist/ 
    method - GET 
    IP - ::ffff:127.0.0.1
   {"timestamp":"2022-02-01T05:48:33.466Z"}
error: 
    status - 500 
    message - ReferenceError: decoded is not defined
    at _callee$ (D:\Book_Store_App\Book-Store\src\middlewares\/auth.middleware.js:28:28)
    at tryCatch (D:\Book_Store_App\Book-Store\node_modules\regenerator-runtime\runtime.js:63:40)
    at Generator.invoke [as _invoke] (D:\Book_Store_App\Book-Store\node_modules\regenerator-runtime\runtime.js:294:22)
    at Generator.next (D:\Book_Store_App\Book-Store\node_modules\regenerator-runtime\runtime.js:119:21)
    at asyncGeneratorStep (D:\Book_Store_App\Book-Store\node_modules\@babel\runtime\helpers\asyncToGenerator.js:3:24)
    at _next (D:\Book_Store_App\Book-Store\node_modules\@babel\runtime\helpers\asyncToGenerator.js:25:9)
    at D:\Book_Store_App\Book-Store\node_modules\@babel\runtime\helpers\asyncToGenerator.js:32:7
    at new Promise (<anonymous>)
    at D:\Book_Store_App\Book-Store\node_modules\@babel\runtime\helpers\asyncToGenerator.js:21:12
    at D:\Book_Store_App\Book-Store\src\middlewares\/auth.middleware.js:15:3 
    url - /api/v1/wishlist/ 
    method - GET 
    IP - ::ffff:127.0.0.1
   {"timestamp":"2022-02-01T05:48:33.667Z"}
error: 
    status - 500 
    message - ReferenceError: decoded is not defined
    at _callee$ (D:\Book_Store_App\Book-Store\src\middlewares\/auth.middleware.js:28:28)
    at tryCatch (D:\Book_Store_App\Book-Store\node_modules\regenerator-runtime\runtime.js:63:40)
    at Generator.invoke [as _invoke] (D:\Book_Store_App\Book-Store\node_modules\regenerator-runtime\runtime.js:294:22)
    at Generator.next (D:\Book_Store_App\Book-Store\node_modules\regenerator-runtime\runtime.js:119:21)
    at asyncGeneratorStep (D:\Book_Store_App\Book-Store\node_modules\@babel\runtime\helpers\asyncToGenerator.js:3:24)
    at _next (D:\Book_Store_App\Book-Store\node_modules\@babel\runtime\helpers\asyncToGenerator.js:25:9)
    at D:\Book_Store_App\Book-Store\node_modules\@babel\runtime\helpers\asyncToGenerator.js:32:7
    at new Promise (<anonymous>)
    at D:\Book_Store_App\Book-Store\node_modules\@babel\runtime\helpers\asyncToGenerator.js:21:12
    at D:\Book_Store_App\Book-Store\src\middlewares\/auth.middleware.js:15:3 
    url - /api/v1/wishlist/ 
    method - GET 
    IP - ::ffff:127.0.0.1
   {"timestamp":"2022-02-01T05:48:33.843Z"}
info: Server started at http://localhost:3000/api/v1/ {"timestamp":"2022-02-01T05:48:59.834Z"}
info: Connected to the database. {"timestamp":"2022-02-01T05:48:59.860Z"}
error: 
    status - 500 
    message - ReferenceError: decoded is not defined
    at _callee$ (D:\Book_Store_App\Book-Store\src\middlewares\/auth.middleware.js:29:7)
    at tryCatch (D:\Book_Store_App\Book-Store\node_modules\regenerator-runtime\runtime.js:63:40)
    at Generator.invoke [as _invoke] (D:\Book_Store_App\Book-Store\node_modules\regenerator-runtime\runtime.js:294:22)
    at Generator.next (D:\Book_Store_App\Book-Store\node_modules\regenerator-runtime\runtime.js:119:21)
    at asyncGeneratorStep (D:\Book_Store_App\Book-Store\node_modules\@babel\runtime\helpers\asyncToGenerator.js:3:24)
    at _next (D:\Book_Store_App\Book-Store\node_modules\@babel\runtime\helpers\asyncToGenerator.js:25:9)
    at D:\Book_Store_App\Book-Store\node_modules\@babel\runtime\helpers\asyncToGenerator.js:32:7
    at new Promise (<anonymous>)
    at D:\Book_Store_App\Book-Store\node_modules\@babel\runtime\helpers\asyncToGenerator.js:21:12
    at D:\Book_Store_App\Book-Store\src\middlewares\/auth.middleware.js:15:3 
    url - /api/v1/wishlist/ 
    method - GET 
    IP - ::ffff:127.0.0.1
   {"timestamp":"2022-02-01T05:49:14.569Z"}
info: Server started at http://localhost:3000/api/v1/ {"timestamp":"2022-02-01T05:49:56.988Z"}
info: Connected to the database. {"timestamp":"2022-02-01T05:49:57.026Z"}
info: Server started at http://localhost:3000/api/v1/ {"timestamp":"2022-02-01T05:50:04.774Z"}
info: Connected to the database. {"timestamp":"2022-02-01T05:50:04.819Z"}
info: Server started at http://localhost:3000/api/v1/ {"timestamp":"2022-02-01T05:50:22.644Z"}
info: Connected to the database. {"timestamp":"2022-02-01T05:50:22.676Z"}
info: Server started at http://localhost:3000/api/v1/ {"timestamp":"2022-02-01T05:50:50.696Z"}
info: Connected to the database. {"timestamp":"2022-02-01T05:50:50.754Z"}
info: Server started at http://localhost:3000/api/v1/ {"timestamp":"2022-02-01T05:50:58.078Z"}
info: Connected to the database. {"timestamp":"2022-02-01T05:50:58.129Z"}
info: Server started at http://localhost:3000/api/v1/ {"timestamp":"2022-02-01T07:06:38.004Z"}
info: Connected to the database. {"timestamp":"2022-02-01T07:06:38.036Z"}
info: Server started at http://localhost:3000/api/v1/ {"timestamp":"2022-02-01T07:07:26.489Z"}
info: Connected to the database. {"timestamp":"2022-02-01T07:07:26.518Z"}
info: Server started at http://localhost:3000/api/v1/ {"timestamp":"2022-02-01T07:08:17.882Z"}
info: Connected to the database. {"timestamp":"2022-02-01T07:08:17.936Z"}
info: Server started at http://localhost:3000/api/v1/ {"timestamp":"2022-02-01T09:26:17.358Z"}
info: Connected to the database. {"timestamp":"2022-02-01T09:26:17.441Z"}
info: Server started at http://localhost:3000/api/v1/ {"timestamp":"2022-02-01T09:27:56.210Z"}
info: Connected to the database. {"timestamp":"2022-02-01T09:27:56.235Z"}
info: Server started at http://localhost:3000/api/v1/ {"timestamp":"2022-02-01T09:28:26.722Z"}
info: Connected to the database. {"timestamp":"2022-02-01T09:28:26.745Z"}
info: Server started at http://localhost:3000/api/v1/ {"timestamp":"2022-02-01T09:29:10.539Z"}
info: Connected to the database. {"timestamp":"2022-02-01T09:29:10.573Z"}
info: Server started at http://localhost:3000/api/v1/ {"timestamp":"2022-02-01T09:29:57.028Z"}
info: Connected to the database. {"timestamp":"2022-02-01T09:29:57.056Z"}
info: Server started at http://localhost:3000/api/v1/ {"timestamp":"2022-02-01T09:31:45.993Z"}
info: Connected to the database. {"timestamp":"2022-02-01T09:31:46.017Z"}
info: Server started at http://localhost:3000/api/v1/ {"timestamp":"2022-02-01T09:32:11.296Z"}
info: Connected to the database. {"timestamp":"2022-02-01T09:32:11.319Z"}
error: 
      status - 400
      message - Authorization token is required 
      url - /api/v1/wishlist/ 
      method - GET 
      IP - ::1
     {"timestamp":"2022-02-01T09:32:21.313Z"}
error: 
    status - 500 
    message - TypeError: Cannot read property '1' of null
    at firstchar (D:\Book_Store_App\Book-Store\node_modules\body-parser\lib\types\json.js:176:37)
    at parse (D:\Book_Store_App\Book-Store\node_modules\body-parser\lib\types\json.js:79:19)
    at D:\Book_Store_App\Book-Store\node_modules\body-parser\lib\read.js:121:18
    at invokeCallback (D:\Book_Store_App\Book-Store\node_modules\raw-body\index.js:224:16)
    at done (D:\Book_Store_App\Book-Store\node_modules\raw-body\index.js:213:7)
    at IncomingMessage.onEnd (D:\Book_Store_App\Book-Store\node_modules\raw-body\index.js:273:7)
    at IncomingMessage.emit (events.js:387:35)
    at IncomingMessage.emit (domain.js:470:12)
    at endReadableNT (internal/streams/readable.js:1317:12)
    at processTicksAndRejections (internal/process/task_queues.js:82:21) 
    url - / 
    method - GET 
    IP - ::1
   {"timestamp":"2022-02-01T09:35:34.879Z"}
info: Server started at http://localhost:3000/api/v1/ {"timestamp":"2022-02-01T09:36:36.196Z"}
info: Connected to the database. {"timestamp":"2022-02-01T09:36:36.223Z"}
info: Server started at http://localhost:3000/api/v1/ {"timestamp":"2022-02-01T09:37:29.790Z"}
info: Connected to the database. {"timestamp":"2022-02-01T09:37:29.814Z"}
info: Server started at http://localhost:3000/api/v1/ {"timestamp":"2022-02-01T09:38:10.576Z"}
info: Connected to the database. {"timestamp":"2022-02-01T09:38:10.601Z"}
error: 
    status - 500 
    message - JsonWebTokenError: jwt must be provided
    at Object.verify (D:\Book_Store_App\Book-Store\node_modules\jsonwebtoken\verify.js:53:17)
    at _callee$ (D:\Book_Store_App\Book-Store\src\middlewares\/auth.middleware.js:24:36)
    at tryCatch (D:\Book_Store_App\Book-Store\node_modules\regenerator-runtime\runtime.js:63:40)
    at Generator.invoke [as _invoke] (D:\Book_Store_App\Book-Store\node_modules\regenerator-runtime\runtime.js:294:22)
    at Generator.next (D:\Book_Store_App\Book-Store\node_modules\regenerator-runtime\runtime.js:119:21)
    at asyncGeneratorStep (D:\Book_Store_App\Book-Store\node_modules\@babel\runtime\helpers\asyncToGenerator.js:3:24)
    at _next (D:\Book_Store_App\Book-Store\node_modules\@babel\runtime\helpers\asyncToGenerator.js:25:9)
    at D:\Book_Store_App\Book-Store\node_modules\@babel\runtime\helpers\asyncToGenerator.js:32:7
    at new Promise (<anonymous>)
    at D:\Book_Store_App\Book-Store\node_modules\@babel\runtime\helpers\asyncToGenerator.js:21:12 
    url - /api/v1/wishlist/ 
    method - GET 
    IP - ::1
   {"timestamp":"2022-02-01T09:39:03.091Z"}
error: 
    status - 500 
    message - JsonWebTokenError: jwt must be provided
    at Object.verify (D:\Book_Store_App\Book-Store\node_modules\jsonwebtoken\verify.js:53:17)
    at _callee$ (D:\Book_Store_App\Book-Store\src\middlewares\/auth.middleware.js:24:36)
    at tryCatch (D:\Book_Store_App\Book-Store\node_modules\regenerator-runtime\runtime.js:63:40)
    at Generator.invoke [as _invoke] (D:\Book_Store_App\Book-Store\node_modules\regenerator-runtime\runtime.js:294:22)
    at Generator.next (D:\Book_Store_App\Book-Store\node_modules\regenerator-runtime\runtime.js:119:21)
    at asyncGeneratorStep (D:\Book_Store_App\Book-Store\node_modules\@babel\runtime\helpers\asyncToGenerator.js:3:24)
    at _next (D:\Book_Store_App\Book-Store\node_modules\@babel\runtime\helpers\asyncToGenerator.js:25:9)
    at D:\Book_Store_App\Book-Store\node_modules\@babel\runtime\helpers\asyncToGenerator.js:32:7
    at new Promise (<anonymous>)
    at D:\Book_Store_App\Book-Store\node_modules\@babel\runtime\helpers\asyncToGenerator.js:21:12 
    url - /api/v1/books/ 
    method - POST 
    IP - ::1
   {"timestamp":"2022-02-01T09:41:04.746Z"}
error: 
    status - 500 
    message - JsonWebTokenError: invalid signature
    at D:\Book_Store_App\Book-Store\node_modules\jsonwebtoken\verify.js:133:19
    at getSecret (D:\Book_Store_App\Book-Store\node_modules\jsonwebtoken\verify.js:90:14)
    at Object.verify (D:\Book_Store_App\Book-Store\node_modules\jsonwebtoken\verify.js:94:10)
    at _callee$ (D:\Book_Store_App\Book-Store\src\middlewares\/auth.middleware.js:24:36)
    at tryCatch (D:\Book_Store_App\Book-Store\node_modules\regenerator-runtime\runtime.js:63:40)
    at Generator.invoke [as _invoke] (D:\Book_Store_App\Book-Store\node_modules\regenerator-runtime\runtime.js:294:22)
    at Generator.next (D:\Book_Store_App\Book-Store\node_modules\regenerator-runtime\runtime.js:119:21)
    at asyncGeneratorStep (D:\Book_Store_App\Book-Store\node_modules\@babel\runtime\helpers\asyncToGenerator.js:3:24)
    at _next (D:\Book_Store_App\Book-Store\node_modules\@babel\runtime\helpers\asyncToGenerator.js:25:9)
    at D:\Book_Store_App\Book-Store\node_modules\@babel\runtime\helpers\asyncToGenerator.js:32:7 
    url - /api/v1/books/ 
    method - POST 
    IP - ::1
   {"timestamp":"2022-02-01T09:41:45.949Z"}
error: 
    status - 500 
    message - JsonWebTokenError: invalid signature
    at D:\Book_Store_App\Book-Store\node_modules\jsonwebtoken\verify.js:133:19
    at getSecret (D:\Book_Store_App\Book-Store\node_modules\jsonwebtoken\verify.js:90:14)
    at Object.verify (D:\Book_Store_App\Book-Store\node_modules\jsonwebtoken\verify.js:94:10)
    at _callee$ (D:\Book_Store_App\Book-Store\src\middlewares\/auth.middleware.js:24:36)
    at tryCatch (D:\Book_Store_App\Book-Store\node_modules\regenerator-runtime\runtime.js:63:40)
    at Generator.invoke [as _invoke] (D:\Book_Store_App\Book-Store\node_modules\regenerator-runtime\runtime.js:294:22)
    at Generator.next (D:\Book_Store_App\Book-Store\node_modules\regenerator-runtime\runtime.js:119:21)
    at asyncGeneratorStep (D:\Book_Store_App\Book-Store\node_modules\@babel\runtime\helpers\asyncToGenerator.js:3:24)
    at _next (D:\Book_Store_App\Book-Store\node_modules\@babel\runtime\helpers\asyncToGenerator.js:25:9)
    at D:\Book_Store_App\Book-Store\node_modules\@babel\runtime\helpers\asyncToGenerator.js:32:7 
    url - /api/v1/wishlist/ 
    method - GET 
    IP - ::1
   {"timestamp":"2022-02-01T09:42:07.213Z"}
info: Server started at http://localhost:3000/api/v1/ {"timestamp":"2022-02-01T09:43:37.118Z"}
info: Connected to the database. {"timestamp":"2022-02-01T09:43:37.141Z"}
info: Server started at http://localhost:3000/api/v1/ {"timestamp":"2022-02-01T09:43:46.593Z"}
info: Connected to the database. {"timestamp":"2022-02-01T09:43:46.629Z"}
info: Server started at http://localhost:3000/api/v1/ {"timestamp":"2022-02-01T09:43:57.248Z"}
info: Connected to the database. {"timestamp":"2022-02-01T09:43:57.343Z"}
info: Server started at http://localhost:3000/api/v1/ {"timestamp":"2022-02-01T09:45:20.018Z"}
info: Connected to the database. {"timestamp":"2022-02-01T09:45:20.054Z"}
info: Server started at http://localhost:3000/api/v1/ {"timestamp":"2022-02-01T09:48:09.085Z"}
info: Connected to the database. {"timestamp":"2022-02-01T09:48:09.134Z"}
error: 
    status - 500 
    message - Error: User Acess Denied
    at _callee$ (D:\Book_Store_App\Book-Store\src\middlewares\/auth.middleware.js:30:14)
    at tryCatch (D:\Book_Store_App\Book-Store\node_modules\regenerator-runtime\runtime.js:63:40)
    at Generator.invoke [as _invoke] (D:\Book_Store_App\Book-Store\node_modules\regenerator-runtime\runtime.js:294:22)
    at Generator.next (D:\Book_Store_App\Book-Store\node_modules\regenerator-runtime\runtime.js:119:21)
    at asyncGeneratorStep (D:\Book_Store_App\Book-Store\node_modules\@babel\runtime\helpers\asyncToGenerator.js:3:24)
    at _next (D:\Book_Store_App\Book-Store\node_modules\@babel\runtime\helpers\asyncToGenerator.js:25:9)
    at D:\Book_Store_App\Book-Store\node_modules\@babel\runtime\helpers\asyncToGenerator.js:32:7
    at new Promise (<anonymous>) 
    url - /api/v1/books/ 
    method - POST 
    IP - ::1
   {"timestamp":"2022-02-01T09:48:58.082Z"}
info: Server started at http://localhost:3000/api/v1/ {"timestamp":"2022-02-01T11:03:02.841Z"}
info: Connected to the database. {"timestamp":"2022-02-01T11:03:02.872Z"}
info: Server started at http://localhost:3000/api/v1/ {"timestamp":"2022-02-01T11:03:33.816Z"}
info: Connected to the database. {"timestamp":"2022-02-01T11:03:33.843Z"}
info: Server started at http://localhost:3000/api/v1/ {"timestamp":"2022-02-01T11:03:43.228Z"}
info: Connected to the database. {"timestamp":"2022-02-01T11:03:43.263Z"}
info: Server started at http://localhost:3000/api/v1/ {"timestamp":"2022-02-01T11:05:35.196Z"}
info: Connected to the database. {"timestamp":"2022-02-01T11:05:35.240Z"}
info: Server started at http://localhost:3000/api/v1/ {"timestamp":"2022-02-01T11:05:42.169Z"}
info: Connected to the database. {"timestamp":"2022-02-01T11:05:42.197Z"}
info: Server started at http://localhost:3000/api/v1/ {"timestamp":"2022-02-01T11:06:05.843Z"}
info: Connected to the database. {"timestamp":"2022-02-01T11:06:05.873Z"}
info: Server started at http://localhost:3000/api/v1/ {"timestamp":"2022-02-01T11:09:48.763Z"}
info: Connected to the database. {"timestamp":"2022-02-01T11:09:48.789Z"}
info: Server started at http://localhost:3000/api/v1/ {"timestamp":"2022-02-01T11:10:15.720Z"}
info: Connected to the database. {"timestamp":"2022-02-01T11:10:15.763Z"}
info: Server started at http://localhost:3000/api/v1/ {"timestamp":"2022-02-01T11:10:40.153Z"}
info: Connected to the database. {"timestamp":"2022-02-01T11:10:40.188Z"}
info: Server started at http://localhost:3000/api/v1/ {"timestamp":"2022-02-01T11:11:26.964Z"}
info: Connected to the database. {"timestamp":"2022-02-01T11:11:26.998Z"}
info: Server started at http://localhost:3000/api/v1/ {"timestamp":"2022-02-01T11:11:49.695Z"}
info: Connected to the database. {"timestamp":"2022-02-01T11:11:49.754Z"}
info: Server started at http://localhost:3000/api/v1/ {"timestamp":"2022-02-01T11:11:59.995Z"}
info: Connected to the database. {"timestamp":"2022-02-01T11:12:00.033Z"}
info: Server started at http://localhost:3000/api/v1/ {"timestamp":"2022-02-01T11:14:09.580Z"}
info: Connected to the database. {"timestamp":"2022-02-01T11:14:09.618Z"}
info: Server started at http://localhost:3000/api/v1/ {"timestamp":"2022-02-01T11:14:17.791Z"}
info: Connected to the database. {"timestamp":"2022-02-01T11:14:17.825Z"}
info: Server started at http://localhost:3000/api/v1/ {"timestamp":"2022-02-01T11:14:28.470Z"}
info: Connected to the database. {"timestamp":"2022-02-01T11:14:28.503Z"}
info: Server started at http://localhost:3000/api/v1/ {"timestamp":"2022-02-01T11:15:00.042Z"}
info: Connected to the database. {"timestamp":"2022-02-01T11:15:00.075Z"}
info: Server started at http://localhost:3000/api/v1/ {"timestamp":"2022-02-01T11:15:13.868Z"}
info: Connected to the database. {"timestamp":"2022-02-01T11:15:13.893Z"}
info: Server started at http://localhost:3000/api/v1/ {"timestamp":"2022-02-01T11:15:33.330Z"}
info: Connected to the database. {"timestamp":"2022-02-01T11:15:33.354Z"}
info: Server started at http://localhost:3000/api/v1/ {"timestamp":"2022-02-01T11:16:18.421Z"}
info: Connected to the database. {"timestamp":"2022-02-01T11:16:18.455Z"}
info: Server started at http://localhost:3000/api/v1/ {"timestamp":"2022-02-01T11:17:41.548Z"}
info: Connected to the database. {"timestamp":"2022-02-01T11:17:41.581Z"}
info: Server started at http://localhost:3000/api/v1/ {"timestamp":"2022-02-01T11:17:55.393Z"}
info: Connected to the database. {"timestamp":"2022-02-01T11:17:55.424Z"}
info: Server started at http://localhost:3000/api/v1/ {"timestamp":"2022-02-01T11:18:10.257Z"}
info: Connected to the database. {"timestamp":"2022-02-01T11:18:10.331Z"}
info: Server started at http://localhost:3000/api/v1/ {"timestamp":"2022-02-01T11:18:40.415Z"}
info: Connected to the database. {"timestamp":"2022-02-01T11:18:40.447Z"}
info: Server started at http://localhost:3000/api/v1/ {"timestamp":"2022-02-01T11:18:51.196Z"}
info: Connected to the database. {"timestamp":"2022-02-01T11:18:51.232Z"}
info: Server started at http://localhost:3000/api/v1/ {"timestamp":"2022-02-01T11:19:10.861Z"}
info: Connected to the database. {"timestamp":"2022-02-01T11:19:10.904Z"}
error: 
    status - 500 
    message - Error: Already Added in Wishlist
    at _callee14$ (D:\Book_Store_App\Book-Store\src\services\/user.service.js:372:15)
    at tryCatch (D:\Book_Store_App\Book-Store\node_modules\regenerator-runtime\runtime.js:63:40)
    at Generator.invoke [as _invoke] (D:\Book_Store_App\Book-Store\node_modules\regenerator-runtime\runtime.js:294:22)
    at Generator.next (D:\Book_Store_App\Book-Store\node_modules\regenerator-runtime\runtime.js:119:21)
    at asyncGeneratorStep (D:\Book_Store_App\Book-Store\node_modules\@babel\runtime\helpers\asyncToGenerator.js:3:24)
    at _next (D:\Book_Store_App\Book-Store\node_modules\@babel\runtime\helpers\asyncToGenerator.js:25:9)
    at processTicksAndRejections (internal/process/task_queues.js:95:5) 
    url - /api/v1/wishlist/61f64042a99dbd428ca5261c 
    method - POST 
    IP - ::1
   {"timestamp":"2022-02-01T11:21:11.669Z"}
error: 
    status - 500 
    message - Error: Book Not in wishlist
    at _callee15$ (D:\Book_Store_App\Book-Store\src\services\/user.service.js:405:9)
    at tryCatch (D:\Book_Store_App\Book-Store\node_modules\regenerator-runtime\runtime.js:63:40)
    at Generator.invoke [as _invoke] (D:\Book_Store_App\Book-Store\node_modules\regenerator-runtime\runtime.js:294:22)
    at Generator.next (D:\Book_Store_App\Book-Store\node_modules\regenerator-runtime\runtime.js:119:21)
    at asyncGeneratorStep (D:\Book_Store_App\Book-Store\node_modules\@babel\runtime\helpers\asyncToGenerator.js:3:24)
    at _next (D:\Book_Store_App\Book-Store\node_modules\@babel\runtime\helpers\asyncToGenerator.js:25:9)
    at processTicksAndRejections (internal/process/task_queues.js:95:5) 
    url - /api/v1/wishlist/61f64062a99dbd428ca52620 
    method - PATCH 
    IP - ::1
   {"timestamp":"2022-02-01T11:22:00.450Z"}
error: 
    status - 500 
    message - Error: Already Added in Wishlist
    at _callee14$ (D:\Book_Store_App\Book-Store\src\services\/user.service.js:372:15)
    at tryCatch (D:\Book_Store_App\Book-Store\node_modules\regenerator-runtime\runtime.js:63:40)
    at Generator.invoke [as _invoke] (D:\Book_Store_App\Book-Store\node_modules\regenerator-runtime\runtime.js:294:22)
    at Generator.next (D:\Book_Store_App\Book-Store\node_modules\regenerator-runtime\runtime.js:119:21)
    at asyncGeneratorStep (D:\Book_Store_App\Book-Store\node_modules\@babel\runtime\helpers\asyncToGenerator.js:3:24)
    at _next (D:\Book_Store_App\Book-Store\node_modules\@babel\runtime\helpers\asyncToGenerator.js:25:9)
    at processTicksAndRejections (internal/process/task_queues.js:95:5) 
    url - /api/v1/wishlist/61f64062a99dbd428ca52620 
    method - POST 
    IP - ::1
   {"timestamp":"2022-02-01T11:22:23.330Z"}
info: Server started at http://localhost:3000/api/v1/ {"timestamp":"2022-02-01T11:26:30.394Z"}
info: Connected to the database. {"timestamp":"2022-02-01T11:26:30.434Z"}
info: Server started at http://localhost:3000/api/v1/ {"timestamp":"2022-02-01T11:36:23.000Z"}
info: Connected to the database. {"timestamp":"2022-02-01T11:36:23.050Z"}
error: 
    status - 500 
    message - Error: User Already Exsist
    at _callee$ (D:\Book_Store_App\Book-Store\src\services\/user.service.js:23:11)
    at tryCatch (D:\Book_Store_App\Book-Store\node_modules\regenerator-runtime\runtime.js:63:40)
    at Generator.invoke [as _invoke] (D:\Book_Store_App\Book-Store\node_modules\regenerator-runtime\runtime.js:294:22)
    at Generator.next (D:\Book_Store_App\Book-Store\node_modules\regenerator-runtime\runtime.js:119:21)
    at asyncGeneratorStep (D:\Book_Store_App\Book-Store\node_modules\@babel\runtime\helpers\asyncToGenerator.js:3:24)
    at _next (D:\Book_Store_App\Book-Store\node_modules\@babel\runtime\helpers\asyncToGenerator.js:25:9)
    at processTicksAndRejections (internal/process/task_queues.js:95:5) 
    url - /api/v1/users/user_registration 
    method - POST 
    IP - ::ffff:127.0.0.1
   {"timestamp":"2022-02-01T11:36:23.272Z"}
error: 
    status - 500 
    message - Error: User does not exist
    at _callee$ (D:\Book_Store_App\Book-Store\src\middlewares\/checkuser.middleware.js:17:10)
    at tryCatch (D:\Book_Store_App\Book-Store\node_modules\regenerator-runtime\runtime.js:63:40)
    at Generator.invoke [as _invoke] (D:\Book_Store_App\Book-Store\node_modules\regenerator-runtime\runtime.js:294:22)
    at Generator.next (D:\Book_Store_App\Book-Store\node_modules\regenerator-runtime\runtime.js:119:21)
    at asyncGeneratorStep (D:\Book_Store_App\Book-Store\node_modules\@babel\runtime\helpers\asyncToGenerator.js:3:24)
    at _next (D:\Book_Store_App\Book-Store\node_modules\@babel\runtime\helpers\asyncToGenerator.js:25:9)
    at processTicksAndRejections (internal/process/task_queues.js:95:5) 
    url - /api/v1/users/user_login 
    method - POST 
    IP - ::ffff:127.0.0.1
   {"timestamp":"2022-02-01T11:36:23.607Z"}
error: 
    status - 500 
    message - Error: Mail id does not exsist
    at _callee3$ (D:\Book_Store_App\Book-Store\src\services\/user.service.js:75:11)
    at tryCatch (D:\Book_Store_App\Book-Store\node_modules\regenerator-runtime\runtime.js:63:40)
    at Generator.invoke [as _invoke] (D:\Book_Store_App\Book-Store\node_modules\regenerator-runtime\runtime.js:294:22)
    at Generator.next (D:\Book_Store_App\Book-Store\node_modules\regenerator-runtime\runtime.js:119:21)
    at asyncGeneratorStep (D:\Book_Store_App\Book-Store\node_modules\@babel\runtime\helpers\asyncToGenerator.js:3:24)
    at _next (D:\Book_Store_App\Book-Store\node_modules\@babel\runtime\helpers\asyncToGenerator.js:25:9)
    at processTicksAndRejections (internal/process/task_queues.js:95:5) 
    url - /api/v1/users/forgetpassword 
    method - POST 
    IP - ::ffff:127.0.0.1
   {"timestamp":"2022-02-01T11:36:23.737Z"}
error: 
    status - 500 
    message - JsonWebTokenError: jwt must be provided
    at Object.verify (D:\Book_Store_App\Book-Store\node_modules\jsonwebtoken\verify.js:53:17)
    at _callee4$ (D:\Book_Store_App\Book-Store\src\services\/user.service.js:81:29)
    at tryCatch (D:\Book_Store_App\Book-Store\node_modules\regenerator-runtime\runtime.js:63:40)
    at Generator.invoke [as _invoke] (D:\Book_Store_App\Book-Store\node_modules\regenerator-runtime\runtime.js:294:22)
    at Generator.next (D:\Book_Store_App\Book-Store\node_modules\regenerator-runtime\runtime.js:119:21)
    at asyncGeneratorStep (D:\Book_Store_App\Book-Store\node_modules\@babel\runtime\helpers\asyncToGenerator.js:3:24)
    at _next (D:\Book_Store_App\Book-Store\node_modules\@babel\runtime\helpers\asyncToGenerator.js:25:9)
    at D:\Book_Store_App\Book-Store\node_modules\@babel\runtime\helpers\asyncToGenerator.js:32:7
    at new Promise (<anonymous>)
    at Object.<anonymous> (D:\Book_Store_App\Book-Store\node_modules\@babel\runtime\helpers\asyncToGenerator.js:21:12)
    at Object.resetPassword (D:\Book_Store_App\Book-Store\src\services\/user.service.js:79:27)
    at _callee4$ (D:\Book_Store_App\Book-Store\src\controllers\/user.controller.js:55:36)
    at tryCatch (D:\Book_Store_App\Book-Store\node_modules\regenerator-runtime\runtime.js:63:40)
    at Generator.invoke [as _invoke] (D:\Book_Store_App\Book-Store\node_modules\regenerator-runtime\runtime.js:294:22)
    at Generator.next (D:\Book_Store_App\Book-Store\node_modules\regenerator-runtime\runtime.js:119:21)
    at asyncGeneratorStep (D:\Book_Store_App\Book-Store\node_modules\@babel\runtime\helpers\asyncToGenerator.js:3:24)
    at _next (D:\Book_Store_App\Book-Store\node_modules\@babel\runtime\helpers\asyncToGenerator.js:25:9)
    at D:\Book_Store_App\Book-Store\node_modules\@babel\runtime\helpers\asyncToGenerator.js:32:7
    at new Promise (<anonymous>)
    at D:\Book_Store_App\Book-Store\node_modules\@babel\runtime\helpers\asyncToGenerator.js:21:12
    at resetPassword (D:\Book_Store_App\Book-Store\src\controllers\/user.controller.js:53:27)
    at Layer.handle [as handle_request] (D:\Book_Store_App\Book-Store\node_modules\express\lib\router\layer.js:95:5)
    at next (D:\Book_Store_App\Book-Store\node_modules\express\lib\router\route.js:137:13)
    at Route.dispatch (D:\Book_Store_App\Book-Store\node_modules\express\lib\router\route.js:112:3)
    at Layer.handle [as handle_request] (D:\Book_Store_App\Book-Store\node_modules\express\lib\router\layer.js:95:5)
    at D:\Book_Store_App\Book-Store\node_modules\express\lib\router\index.js:281:22
    at Function.process_params (D:\Book_Store_App\Book-Store\node_modules\express\lib\router\index.js:341:12)
    at next (D:\Book_Store_App\Book-Store\node_modules\express\lib\router\index.js:275:10)
    at Function.handle (D:\Book_Store_App\Book-Store\node_modules\express\lib\router\index.js:174:3)
    at router (D:\Book_Store_App\Book-Store\node_modules\express\lib\router\index.js:47:12)
    at Layer.handle [as handle_request] (D:\Book_Store_App\Book-Store\node_modules\express\lib\router\layer.js:95:5)
    at trim_prefix (D:\Book_Store_App\Book-Store\node_modules\express\lib\router\index.js:323:13)
    at D:\Book_Store_App\Book-Store\node_modules\express\lib\router\index.js:284:7
    at Function.process_params (D:\Book_Store_App\Book-Store\node_modules\express\lib\router\index.js:341:12)
    at next (D:\Book_Store_App\Book-Store\node_modules\express\lib\router\index.js:275:10)
    at Function.handle (D:\Book_Store_App\Book-Store\node_modules\express\lib\router\index.js:174:3)
    at router (D:\Book_Store_App\Book-Store\node_modules\express\lib\router\index.js:47:12)
    at Layer.handle [as handle_request] (D:\Book_Store_App\Book-Store\node_modules\express\lib\router\layer.js:95:5)
    at trim_prefix (D:\Book_Store_App\Book-Store\node_modules\express\lib\router\index.js:323:13)
    at D:\Book_Store_App\Book-Store\node_modules\express\lib\router\index.js:284:7
    at Function.process_params (D:\Book_Store_App\Book-Store\node_modules\express\lib\router\index.js:341:12)
    at next (D:\Book_Store_App\Book-Store\node_modules\express\lib\router\index.js:275:10)
    at logger (D:\Book_Store_App\Book-Store\node_modules\morgan\index.js:144:5)
    at Layer.handle [as handle_request] (D:\Book_Store_App\Book-Store\node_modules\express\lib\router\layer.js:95:5)
    at trim_prefix (D:\Book_Store_App\Book-Store\node_modules\express\lib\router\index.js:323:13)
    at D:\Book_Store_App\Book-Store\node_modules\express\lib\router\index.js:284:7
    at Function.process_params (D:\Book_Store_App\Book-Store\node_modules\express\lib\router\index.js:341:12)
    at next (D:\Book_Store_App\Book-Store\node_modules\express\lib\router\index.js:275:10)
    at D:\Book_Store_App\Book-Store\node_modules\body-parser\lib\read.js:130:5
    at invokeCallback (D:\Book_Store_App\Book-Store\node_modules\raw-body\index.js:224:16)
    at done (D:\Book_Store_App\Book-Store\node_modules\raw-body\index.js:213:7)
    at IncomingMessage.onEnd (D:\Book_Store_App\Book-Store\node_modules\raw-body\index.js:273:7)
    at IncomingMessage.emit (events.js:387:35)
    at endReadableNT (internal/streams/readable.js:1317:12)
    at processTicksAndRejections (internal/process/task_queues.js:82:21) 
    url - /api/v1/users/resetpassword 
    method - POST 
    IP - ::ffff:127.0.0.1
   {"timestamp":"2022-02-01T11:36:23.794Z"}
info: Server started at http://localhost:3000/api/v1/ {"timestamp":"2022-02-01T11:37:31.015Z"}
info: Connected to the database. {"timestamp":"2022-02-01T11:37:31.108Z"}
info: Server started at http://localhost:3000/api/v1/ {"timestamp":"2022-02-01T11:38:54.131Z"}
info: Connected to the database. {"timestamp":"2022-02-01T11:38:54.242Z"}
info: Server started at http://localhost:3000/api/v1/ {"timestamp":"2022-02-01T11:40:37.860Z"}
info: Connected to the database. {"timestamp":"2022-02-01T11:40:37.887Z"}
info: Server started at http://localhost:3000/api/v1/ {"timestamp":"2022-02-01T11:51:30.540Z"}
info: Connected to the database. {"timestamp":"2022-02-01T11:51:30.633Z"}
info: Server started at http://localhost:3000/api/v1/ {"timestamp":"2022-02-01T11:51:44.998Z"}
info: Connected to the database. {"timestamp":"2022-02-01T11:51:45.023Z"}
info: Server started at http://localhost:3000/api/v1/ {"timestamp":"2022-02-01T11:52:18.878Z"}
info: Connected to the database. {"timestamp":"2022-02-01T11:52:18.903Z"}
info: Server started at http://localhost:3000/api/v1/ {"timestamp":"2022-02-01T11:52:54.136Z"}
info: Connected to the database. {"timestamp":"2022-02-01T11:52:54.229Z"}
info: Server started at http://localhost:3000/api/v1/ {"timestamp":"2022-02-01T12:15:03.976Z"}
info: Connected to the database. {"timestamp":"2022-02-01T12:15:04.045Z"}
error: 
    status - 500 
    message - Error: Wishlist does not created
    at _callee16$ (D:\Book_Store_App\Book-Store\src\services\/user.service.js:422:12)
    at tryCatch (D:\Book_Store_App\Book-Store\node_modules\regenerator-runtime\runtime.js:63:40)
    at Generator.invoke [as _invoke] (D:\Book_Store_App\Book-Store\node_modules\regenerator-runtime\runtime.js:294:22)
    at Generator.next (D:\Book_Store_App\Book-Store\node_modules\regenerator-runtime\runtime.js:119:21)
    at asyncGeneratorStep (D:\Book_Store_App\Book-Store\node_modules\@babel\runtime\helpers\asyncToGenerator.js:3:24)
    at _next (D:\Book_Store_App\Book-Store\node_modules\@babel\runtime\helpers\asyncToGenerator.js:25:9)
    at processTicksAndRejections (internal/process/task_queues.js:95:5) 
    url - /api/v1/wishlist/ 
    method - GET 
    IP - ::ffff:127.0.0.1
   {"timestamp":"2022-02-01T12:15:05.717Z"}
info: Server started at http://localhost:3000/api/v1/ {"timestamp":"2022-02-01T12:16:29.176Z"}
info: Connected to the database. {"timestamp":"2022-02-01T12:16:29.260Z"}
error: 
    status - 500 
    message - Error: Wishlist does not created
    at _callee16$ (D:\Book_Store_App\Book-Store\src\services\/user.service.js:422:12)
    at tryCatch (D:\Book_Store_App\Book-Store\node_modules\regenerator-runtime\runtime.js:63:40)
    at Generator.invoke [as _invoke] (D:\Book_Store_App\Book-Store\node_modules\regenerator-runtime\runtime.js:294:22)
    at Generator.next (D:\Book_Store_App\Book-Store\node_modules\regenerator-runtime\runtime.js:119:21)
    at asyncGeneratorStep (D:\Book_Store_App\Book-Store\node_modules\@babel\runtime\helpers\asyncToGenerator.js:3:24)
    at _next (D:\Book_Store_App\Book-Store\node_modules\@babel\runtime\helpers\asyncToGenerator.js:25:9)
    at processTicksAndRejections (internal/process/task_queues.js:95:5) 
    url - /api/v1/wishlist/ 
    method - GET 
    IP - ::ffff:127.0.0.1
   {"timestamp":"2022-02-01T12:16:30.345Z"}
info: Server started at http://localhost:3000/api/v1/ {"timestamp":"2022-02-01T12:23:24.274Z"}
info: Connected to the database. {"timestamp":"2022-02-01T12:23:24.388Z"}
error: 
    status - 500 
    message - CastError: Cast to ObjectId failed for value "4{BookID}" (type string) at path "_id" for model "Book"
    at model.Query.exec (D:\Book_Store_App\Book-Store\node_modules\mongoose\lib\query.js:4498:21)
    at model.Query.Query.then (D:\Book_Store_App\Book-Store\node_modules\mongoose\lib\query.js:4592:15)
    at processTicksAndRejections (internal/process/task_queues.js:95:5) 
    url - /api/v1/wishlist/4%7BBookID%7D 
    method - POST 
    IP - ::ffff:127.0.0.1
   {"timestamp":"2022-02-01T12:23:25.467Z"}
error: 
    status - 500 
    message - CastError: Cast to ObjectId failed for value "{ _id: 'undefined' }" (type Object) at path "_id" for model "Book"
    at model.Query.exec (D:\Book_Store_App\Book-Store\node_modules\mongoose\lib\query.js:4498:21)
    at model.Query.Query.then (D:\Book_Store_App\Book-Store\node_modules\mongoose\lib\query.js:4592:15)
    at processTicksAndRejections (internal/process/task_queues.js:95:5) 
    url - /api/v1/books/undefined 
    method - GET 
    IP - ::ffff:127.0.0.1
   {"timestamp":"2022-02-01T12:23:25.534Z"}
info: Server started at http://localhost:3000/api/v1/ {"timestamp":"2022-02-01T12:24:16.659Z"}
info: Connected to the database. {"timestamp":"2022-02-01T12:24:16.767Z"}
error: 
    status - 500 
    message - CastError: Cast to ObjectId failed for value "4{BookID}" (type string) at path "_id" for model "Book"
    at model.Query.exec (D:\Book_Store_App\Book-Store\node_modules\mongoose\lib\query.js:4498:21)
    at model.Query.Query.then (D:\Book_Store_App\Book-Store\node_modules\mongoose\lib\query.js:4592:15)
    at processTicksAndRejections (internal/process/task_queues.js:95:5) 
    url - /api/v1/wishlist/4%7BBookID%7D 
    method - POST 
    IP - ::ffff:127.0.0.1
   {"timestamp":"2022-02-01T12:24:17.716Z"}
info: Server started at http://localhost:3000/api/v1/ {"timestamp":"2022-02-01T12:26:45.146Z"}
info: Connected to the database. {"timestamp":"2022-02-01T12:26:45.246Z"}
error: 
    status - 500 
    message - CastError: Cast to ObjectId failed for value "undefined" (type string) at path "_id" for model "Book"
    at model.Query.exec (D:\Book_Store_App\Book-Store\node_modules\mongoose\lib\query.js:4498:21)
    at model.Query.Query.then (D:\Book_Store_App\Book-Store\node_modules\mongoose\lib\query.js:4592:15)
    at processTicksAndRejections (internal/process/task_queues.js:95:5) 
    url - /api/v1/wishlist/undefined 
    method - POST 
    IP - ::ffff:127.0.0.1
   {"timestamp":"2022-02-01T12:26:46.219Z"}
error: 
    status - 500 
    message - CastError: Cast to ObjectId failed for value "undefined" (type string) at path "_id" for model "Book"
    at model.Query.exec (D:\Book_Store_App\Book-Store\node_modules\mongoose\lib\query.js:4498:21)
    at model.Query.Query.then (D:\Book_Store_App\Book-Store\node_modules\mongoose\lib\query.js:4592:15)
    at processTicksAndRejections (internal/process/task_queues.js:95:5) 
    url - /api/v1/wishlist/undefined 
    method - POST 
    IP - ::ffff:127.0.0.1
   {"timestamp":"2022-02-01T12:26:46.303Z"}
info: Server started at http://localhost:3000/api/v1/ {"timestamp":"2022-02-01T12:39:05.415Z"}
info: Connected to the database. {"timestamp":"2022-02-01T12:39:05.508Z"}
info: Server started at http://localhost:3000/api/v1/ {"timestamp":"2022-02-01T12:39:58.014Z"}
info: Connected to the database. {"timestamp":"2022-02-01T12:39:58.106Z"}
info: Server started at http://localhost:3000/api/v1/ {"timestamp":"2022-02-01T13:26:40.856Z"}
info: Connected to the database. {"timestamp":"2022-02-01T13:26:41.083Z"}
info: Server started at http://localhost:3000/api/v1/ {"timestamp":"2022-02-01T13:27:08.593Z"}
info: Connected to the database. {"timestamp":"2022-02-01T13:27:08.678Z"}
info: Server started at http://localhost:3000/api/v1/ {"timestamp":"2022-02-01T13:30:31.147Z"}
info: Connected to the database. {"timestamp":"2022-02-01T13:30:31.246Z"}
info: Server started at http://localhost:3000/api/v1/ {"timestamp":"2022-02-01T13:33:50.038Z"}
info: Connected to the database. {"timestamp":"2022-02-01T13:33:50.132Z"}
info: Server started at http://localhost:3000/api/v1/ {"timestamp":"2022-02-01T13:34:55.108Z"}
info: Connected to the database. {"timestamp":"2022-02-01T13:34:55.197Z"}
info: Server started at http://localhost:3000/api/v1/ {"timestamp":"2022-02-01T13:43:55.839Z"}
info: Connected to the database. {"timestamp":"2022-02-01T13:43:55.946Z"}
error: 
    status - 500 
    message - Error: Book Not Available
    at _callee10$ (D:\Book_Store_App\Book-Store\src\services\/user.service.js:268:11)
    at tryCatch (D:\Book_Store_App\Book-Store\node_modules\regenerator-runtime\runtime.js:63:40)
    at Generator.invoke [as _invoke] (D:\Book_Store_App\Book-Store\node_modules\regenerator-runtime\runtime.js:294:22)
    at Generator.next (D:\Book_Store_App\Book-Store\node_modules\regenerator-runtime\runtime.js:119:21)
    at asyncGeneratorStep (D:\Book_Store_App\Book-Store\node_modules\@babel\runtime\helpers\asyncToGenerator.js:3:24)
    at _next (D:\Book_Store_App\Book-Store\node_modules\@babel\runtime\helpers\asyncToGenerator.js:25:9)
    at processTicksAndRejections (internal/process/task_queues.js:95:5) 
    url - /api/v1/cart/ 
    method - POST 
    IP - ::ffff:127.0.0.1
   {"timestamp":"2022-02-01T13:43:56.963Z"}
info: Server started at http://localhost:3000/api/v1/ {"timestamp":"2022-02-01T13:45:57.427Z"}
info: Connected to the database. {"timestamp":"2022-02-01T13:45:57.496Z"}
error: 
    status - 500 
    message - Error: Book Not Available
    at _callee10$ (D:\Book_Store_App\Book-Store\src\services\/user.service.js:268:11)
    at tryCatch (D:\Book_Store_App\Book-Store\node_modules\regenerator-runtime\runtime.js:63:40)
    at Generator.invoke [as _invoke] (D:\Book_Store_App\Book-Store\node_modules\regenerator-runtime\runtime.js:294:22)
    at Generator.next (D:\Book_Store_App\Book-Store\node_modules\regenerator-runtime\runtime.js:119:21)
    at asyncGeneratorStep (D:\Book_Store_App\Book-Store\node_modules\@babel\runtime\helpers\asyncToGenerator.js:3:24)
    at _next (D:\Book_Store_App\Book-Store\node_modules\@babel\runtime\helpers\asyncToGenerator.js:25:9)
    at processTicksAndRejections (internal/process/task_queues.js:95:5) 
    url - /api/v1/cart/ 
    method - POST 
    IP - ::ffff:127.0.0.1
   {"timestamp":"2022-02-01T13:45:58.990Z"}
info: Server started at http://localhost:3000/api/v1/ {"timestamp":"2022-02-01T13:53:26.815Z"}
info: Connected to the database. {"timestamp":"2022-02-01T13:53:26.871Z"}
info: Server started at http://localhost:3000/api/v1/ {"timestamp":"2022-02-01T13:58:15.065Z"}
info: Connected to the database. {"timestamp":"2022-02-01T13:58:15.118Z"}
info: Server started at http://localhost:3000/api/v1/ {"timestamp":"2022-02-01T14:11:47.478Z"}
info: Connected to the database. {"timestamp":"2022-02-01T14:11:47.584Z"}
error: 
    status - 500 
    message - TypeError: Cannot read property 'Book' of null
    at _callee12$ (D:\Book_Store_App\Book-Store\src\services\/user.service.js:301:54)
    at tryCatch (D:\Book_Store_App\Book-Store\node_modules\regenerator-runtime\runtime.js:63:40)
    at Generator.invoke [as _invoke] (D:\Book_Store_App\Book-Store\node_modules\regenerator-runtime\runtime.js:294:22)
    at Generator.next (D:\Book_Store_App\Book-Store\node_modules\regenerator-runtime\runtime.js:119:21)
    at asyncGeneratorStep (D:\Book_Store_App\Book-Store\node_modules\@babel\runtime\helpers\asyncToGenerator.js:3:24)
    at _next (D:\Book_Store_App\Book-Store\node_modules\@babel\runtime\helpers\asyncToGenerator.js:25:9)
    at processTicksAndRejections (internal/process/task_queues.js:95:5) 
    url - /api/v1/cart/ 
    method - PATCH 
    IP - ::ffff:127.0.0.1
   {"timestamp":"2022-02-01T14:11:48.488Z"}
info: Server started at http://localhost:3000/api/v1/ {"timestamp":"2022-02-01T14:14:44.679Z"}
info: Connected to the database. {"timestamp":"2022-02-01T14:14:44.776Z"}
error: 
    status - 500 
    message - TypeError: Cannot read property 'Book' of null
    at _callee12$ (D:\Book_Store_App\Book-Store\src\services\/user.service.js:301:54)
    at tryCatch (D:\Book_Store_App\Book-Store\node_modules\regenerator-runtime\runtime.js:63:40)
    at Generator.invoke [as _invoke] (D:\Book_Store_App\Book-Store\node_modules\regenerator-runtime\runtime.js:294:22)
    at Generator.next (D:\Book_Store_App\Book-Store\node_modules\regenerator-runtime\runtime.js:119:21)
    at asyncGeneratorStep (D:\Book_Store_App\Book-Store\node_modules\@babel\runtime\helpers\asyncToGenerator.js:3:24)
    at _next (D:\Book_Store_App\Book-Store\node_modules\@babel\runtime\helpers\asyncToGenerator.js:25:9)
    at processTicksAndRejections (internal/process/task_queues.js:95:5) 
    url - /api/v1/cart/ 
    method - PATCH 
    IP - ::ffff:127.0.0.1
   {"timestamp":"2022-02-01T14:14:45.664Z"}
info: Server started at http://localhost:3000/api/v1/ {"timestamp":"2022-02-01T14:49:36.043Z"}
info: Connected to the database. {"timestamp":"2022-02-01T14:49:36.142Z"}
error: 
    status - 500 
    message - TypeError: Cannot read property 'Book' of null
    at _callee12$ (D:\Book_Store_App\Book-Store\src\services\/user.service.js:301:54)
    at tryCatch (D:\Book_Store_App\Book-Store\node_modules\regenerator-runtime\runtime.js:63:40)
    at Generator.invoke [as _invoke] (D:\Book_Store_App\Book-Store\node_modules\regenerator-runtime\runtime.js:294:22)
    at Generator.next (D:\Book_Store_App\Book-Store\node_modules\regenerator-runtime\runtime.js:119:21)
    at asyncGeneratorStep (D:\Book_Store_App\Book-Store\node_modules\@babel\runtime\helpers\asyncToGenerator.js:3:24)
    at _next (D:\Book_Store_App\Book-Store\node_modules\@babel\runtime\helpers\asyncToGenerator.js:25:9)
    at processTicksAndRejections (internal/process/task_queues.js:95:5) 
    url - /api/v1/cart/ 
    method - PATCH 
    IP - ::ffff:127.0.0.1
   {"timestamp":"2022-02-01T14:49:37.042Z"}
info: Server started at http://localhost:3000/api/v1/ {"timestamp":"2022-02-01T14:50:41.393Z"}
info: Connected to the database. {"timestamp":"2022-02-01T14:50:41.481Z"}
error: 
    status - 500 
    message - TypeError: Cannot read property 'Book' of null
    at _callee12$ (D:\Book_Store_App\Book-Store\src\services\/user.service.js:301:54)
    at tryCatch (D:\Book_Store_App\Book-Store\node_modules\regenerator-runtime\runtime.js:63:40)
    at Generator.invoke [as _invoke] (D:\Book_Store_App\Book-Store\node_modules\regenerator-runtime\runtime.js:294:22)
    at Generator.next (D:\Book_Store_App\Book-Store\node_modules\regenerator-runtime\runtime.js:119:21)
    at asyncGeneratorStep (D:\Book_Store_App\Book-Store\node_modules\@babel\runtime\helpers\asyncToGenerator.js:3:24)
    at _next (D:\Book_Store_App\Book-Store\node_modules\@babel\runtime\helpers\asyncToGenerator.js:25:9)
    at processTicksAndRejections (internal/process/task_queues.js:95:5) 
    url - /api/v1/cart/ 
    method - PATCH 
    IP - ::ffff:127.0.0.1
   {"timestamp":"2022-02-01T14:50:42.419Z"}
info: Server started at http://localhost:3000/api/v1/ {"timestamp":"2022-02-01T15:06:45.194Z"}
info: Connected to the database. {"timestamp":"2022-02-01T15:06:45.314Z"}
error: 
    status - 500 
    message - TypeError: Cannot read property 'Book' of null
    at _callee12$ (D:\Book_Store_App\Book-Store\src\services\/user.service.js:301:54)
    at tryCatch (D:\Book_Store_App\Book-Store\node_modules\regenerator-runtime\runtime.js:63:40)
    at Generator.invoke [as _invoke] (D:\Book_Store_App\Book-Store\node_modules\regenerator-runtime\runtime.js:294:22)
    at Generator.next (D:\Book_Store_App\Book-Store\node_modules\regenerator-runtime\runtime.js:119:21)
    at asyncGeneratorStep (D:\Book_Store_App\Book-Store\node_modules\@babel\runtime\helpers\asyncToGenerator.js:3:24)
    at _next (D:\Book_Store_App\Book-Store\node_modules\@babel\runtime\helpers\asyncToGenerator.js:25:9)
    at processTicksAndRejections (internal/process/task_queues.js:95:5) 
    url - /api/v1/cart/ 
    method - PATCH 
    IP - ::ffff:127.0.0.1
   {"timestamp":"2022-02-01T15:06:46.664Z"}
info: Server started at http://localhost:3000/api/v1/ {"timestamp":"2022-02-01T15:08:46.301Z"}
info: Connected to the database. {"timestamp":"2022-02-01T15:08:46.399Z"}
error: 
    status - 500 
    message - TypeError: Cannot read property 'Book' of null
    at _callee12$ (D:\Book_Store_App\Book-Store\src\services\/user.service.js:301:54)
    at tryCatch (D:\Book_Store_App\Book-Store\node_modules\regenerator-runtime\runtime.js:63:40)
    at Generator.invoke [as _invoke] (D:\Book_Store_App\Book-Store\node_modules\regenerator-runtime\runtime.js:294:22)
    at Generator.next (D:\Book_Store_App\Book-Store\node_modules\regenerator-runtime\runtime.js:119:21)
    at asyncGeneratorStep (D:\Book_Store_App\Book-Store\node_modules\@babel\runtime\helpers\asyncToGenerator.js:3:24)
    at _next (D:\Book_Store_App\Book-Store\node_modules\@babel\runtime\helpers\asyncToGenerator.js:25:9)
    at processTicksAndRejections (internal/process/task_queues.js:95:5) 
    url - /api/v1/cart/ 
    method - PATCH 
    IP - ::ffff:127.0.0.1
   {"timestamp":"2022-02-01T15:08:47.308Z"}
info: Server started at http://localhost:3000/api/v1/ {"timestamp":"2022-02-01T15:17:50.583Z"}
info: Connected to the database. {"timestamp":"2022-02-01T15:17:50.682Z"}
error: 
    status - 500 
    message - ValidationError: Cart validation failed: Book.0.Total_Price: Cast to Number failed for value "NaN" (type number) at path "Total_Price", TotalAmount: Cast to Number failed for value "NaN" (type number) at path "TotalAmount"
    at model.Document.invalidate (D:\Book_Store_App\Book-Store\node_modules\mongoose\lib\document.js:2782:32)
    at EmbeddedDocument.invalidate (D:\Book_Store_App\Book-Store\node_modules\mongoose\lib\types\embedded.js:298:29)
    at EmbeddedDocument.$set (D:\Book_Store_App\Book-Store\node_modules\mongoose\lib\document.js:1349:12)
    at EmbeddedDocument.$set (D:\Book_Store_App\Book-Store\node_modules\mongoose\lib\document.js:1058:16)
    at EmbeddedDocument.Document (D:\Book_Store_App\Book-Store\node_modules\mongoose\lib\document.js:148:12)
    at EmbeddedDocument [as constructor] (D:\Book_Store_App\Book-Store\node_modules\mongoose\lib\types\embedded.js:47:12)
    at new EmbeddedDocument (D:\Book_Store_App\Book-Store\node_modules\mongoose\lib\schema\documentarray.js:116:17)
    at DocumentArrayPath.cast (D:\Book_Store_App\Book-Store\node_modules\mongoose\lib\schema\documentarray.js:462:22)
    at DocumentArrayPath.cast (D:\Book_Store_App\Book-Store\node_modules\mongoose\lib\schema\documentarray.js:393:17)
    at DocumentArrayPath.SchemaType.applySetters (D:\Book_Store_App\Book-Store\node_modules\mongoose\lib\schematype.js:1123:12)
    at model.$set (D:\Book_Store_App\Book-Store\node_modules\mongoose\lib\document.js:1309:20)
    at model.$set (D:\Book_Store_App\Book-Store\node_modules\mongoose\lib\document.js:1058:16)
    at model.Document (D:\Book_Store_App\Book-Store\node_modules\mongoose\lib\document.js:148:12)
    at model.Model (D:\Book_Store_App\Book-Store\node_modules\mongoose\lib\model.js:106:12)
    at new model (D:\Book_Store_App\Book-Store\node_modules\mongoose\lib\model.js:4911:15)
    at _callee10$ (D:\Book_Store_App\Book-Store\src\services\/user.service.js:185:21)
    at tryCatch (D:\Book_Store_App\Book-Store\node_modules\regenerator-runtime\runtime.js:63:40)
    at Generator.invoke [as _invoke] (D:\Book_Store_App\Book-Store\node_modules\regenerator-runtime\runtime.js:294:22)
    at Generator.next (D:\Book_Store_App\Book-Store\node_modules\regenerator-runtime\runtime.js:119:21)
    at asyncGeneratorStep (D:\Book_Store_App\Book-Store\node_modules\@babel\runtime\helpers\asyncToGenerator.js:3:24)
    at _next (D:\Book_Store_App\Book-Store\node_modules\@babel\runtime\helpers\asyncToGenerator.js:25:9)
    at processTicksAndRejections (internal/process/task_queues.js:95:5) 
    url - /api/v1/cart/ 
    method - POST 
    IP - ::ffff:127.0.0.1
   {"timestamp":"2022-02-01T15:17:51.638Z"}
info: Server started at http://localhost:3000/api/v1/ {"timestamp":"2022-02-01T15:21:42.361Z"}
info: Connected to the database. {"timestamp":"2022-02-01T15:21:42.528Z"}
error: 
    status - 500 
    message - ValidationError: Cart validation failed: Book.0.Total_Price: Cast to Number failed for value "NaN" (type number) at path "Total_Price", TotalAmount: Cast to Number failed for value "NaN" (type number) at path "TotalAmount"
    at model.Document.invalidate (D:\Book_Store_App\Book-Store\node_modules\mongoose\lib\document.js:2782:32)
    at EmbeddedDocument.invalidate (D:\Book_Store_App\Book-Store\node_modules\mongoose\lib\types\embedded.js:298:29)
    at EmbeddedDocument.$set (D:\Book_Store_App\Book-Store\node_modules\mongoose\lib\document.js:1349:12)
    at EmbeddedDocument.$set (D:\Book_Store_App\Book-Store\node_modules\mongoose\lib\document.js:1058:16)
    at EmbeddedDocument.Document (D:\Book_Store_App\Book-Store\node_modules\mongoose\lib\document.js:148:12)
    at EmbeddedDocument [as constructor] (D:\Book_Store_App\Book-Store\node_modules\mongoose\lib\types\embedded.js:47:12)
    at new EmbeddedDocument (D:\Book_Store_App\Book-Store\node_modules\mongoose\lib\schema\documentarray.js:116:17)
    at DocumentArrayPath.cast (D:\Book_Store_App\Book-Store\node_modules\mongoose\lib\schema\documentarray.js:462:22)
    at DocumentArrayPath.cast (D:\Book_Store_App\Book-Store\node_modules\mongoose\lib\schema\documentarray.js:393:17)
    at DocumentArrayPath.SchemaType.applySetters (D:\Book_Store_App\Book-Store\node_modules\mongoose\lib\schematype.js:1123:12)
    at model.$set (D:\Book_Store_App\Book-Store\node_modules\mongoose\lib\document.js:1309:20)
    at model.$set (D:\Book_Store_App\Book-Store\node_modules\mongoose\lib\document.js:1058:16)
    at model.Document (D:\Book_Store_App\Book-Store\node_modules\mongoose\lib\document.js:148:12)
    at model.Model (D:\Book_Store_App\Book-Store\node_modules\mongoose\lib\model.js:106:12)
    at new model (D:\Book_Store_App\Book-Store\node_modules\mongoose\lib\model.js:4911:15)
    at _callee10$ (D:\Book_Store_App\Book-Store\src\services\/user.service.js:185:21)
    at tryCatch (D:\Book_Store_App\Book-Store\node_modules\regenerator-runtime\runtime.js:63:40)
    at Generator.invoke [as _invoke] (D:\Book_Store_App\Book-Store\node_modules\regenerator-runtime\runtime.js:294:22)
    at Generator.next (D:\Book_Store_App\Book-Store\node_modules\regenerator-runtime\runtime.js:119:21)
    at asyncGeneratorStep (D:\Book_Store_App\Book-Store\node_modules\@babel\runtime\helpers\asyncToGenerator.js:3:24)
    at _next (D:\Book_Store_App\Book-Store\node_modules\@babel\runtime\helpers\asyncToGenerator.js:25:9)
    at processTicksAndRejections (internal/process/task_queues.js:95:5) 
    url - /api/v1/cart/ 
    method - POST 
    IP - ::ffff:127.0.0.1
   {"timestamp":"2022-02-01T15:21:43.688Z"}
info: Server started at http://localhost:3000/api/v1/ {"timestamp":"2022-02-01T15:24:42.455Z"}
info: Connected to the database. {"timestamp":"2022-02-01T15:24:42.547Z"}
info: Server started at http://localhost:3000/api/v1/ {"timestamp":"2022-02-01T15:25:43.709Z"}
info: Connected to the database. {"timestamp":"2022-02-01T15:25:43.822Z"}
info: Server started at http://localhost:3000/api/v1/ {"timestamp":"2022-02-01T15:27:18.053Z"}
info: Connected to the database. {"timestamp":"2022-02-01T15:27:18.158Z"}
info: Server started at http://localhost:3000/api/v1/ {"timestamp":"2022-02-01T15:29:31.501Z"}
info: Connected to the database. {"timestamp":"2022-02-01T15:29:31.631Z"}
info: Server started at http://localhost:3000/api/v1/ {"timestamp":"2022-02-01T15:31:27.832Z"}
info: Connected to the database. {"timestamp":"2022-02-01T15:31:28.023Z"}
info: Server started at http://localhost:3000/api/v1/ {"timestamp":"2022-02-01T15:43:23.954Z"}
info: Connected to the database. {"timestamp":"2022-02-01T15:43:24.069Z"}
info: Server started at http://localhost:3000/api/v1/ {"timestamp":"2022-02-01T15:52:25.154Z"}
info: Connected to the database. {"timestamp":"2022-02-01T15:52:25.182Z"}
